samples/deviceCodeSample/src/main/java/deviceCodeFlowMain.java

```diff
@@ -0,0 +1,45 @@
+import com.azure.identity.DeviceCodeCredential;
+import com.azure.identity.DeviceCodeCredentialBuilder;
+import com.microsoft.graph.authentication.TokenCredentialAuthProvider;
+import com.microsoft.graph.httpcore.HttpClients;
+import okhttp3.*;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+
+public class deviceCodeFlowMain {
+
+    //Replace CLIENT_ID with your own client id from an app that is configured according to the requirements below
+    //for requirements visit:
+    //https://github.com/Azure/azure-sdk-for-java/wiki/Set-up-Your-Environment-for-Authentication#enable-applications-for-device-code-flow
+    private final static String CLIENT_ID = "<your client/app id>";
+
+    //Set the scopes for your ms-graph request
+    private final static List<String> SCOPES = Arrays.asList("User.ReadBasic.All", "User.Read");
+
+    public static void main(String[] args) throws Exception {
+        final DeviceCodeCredential deviceCodeCred = new DeviceCodeCredentialBuilder()
+                .clientId(CLIENT_ID)
+                .challengeConsumer(challenge -> {System.out.println(challenge.getMessage());})
```

Comment:
No need for brackets and the semi-colon, i.e. `System.out.println(challenge.getMessage())` is enough.

Reply:
```suggestion
                .challengeConsumer(challenge -> System.out.println(challenge.getMessage()))
```